projects/com.oracle.truffle.llvm.asm.amd64/src/com/oracle/truffle/llvm/asm/amd64/AsmFactory.java

```diff
@@ -1008,6 +1023,168 @@ void createBinaryOperationImplicitSize(String operation, AsmOperand a, AsmOperan
                     throw new AsmParseException("invalid operand type: " + dstType);
                 }
                 break;
+            case "bsr":
```

Comment:
Could we (for the next PR) implement an interface for the creation of inline assembly instructions, similarly as we do in the `NodeFactory`?